tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -53,6 +53,24 @@
 raw_od.plot(n_channels=len(raw_od.ch_names), duration=500)
 
 
+###############################################################################
+# Evaluating the quality of the data
+# ----------------------------------
+#
+# At this stage we can quantify the quality of the coupling
+# between the scalp and the optodes using the scalp coupling index. This
+# method looks at the presence of a prominent synchronous signal in the
+# frequency range of cardiac signals across both photodetected signals.
+#
+# As this data is clean and the coupling is good for all channels we will
```

Comment:
"for all channels (e.g., R > 0.5) we will"

if you know of or feel comfortable giving some rule of thumb (I have no idea what the guideline is, if there is one)